guide/phone_order_delivery_service/phone_order_delivery_service.bal

```diff
@@ -0,0 +1,218 @@
+
+//json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+//@kubernetes:Ingress {
+//hostname:"ballerina.guides.io",
+//name:"ballerina-guides-phone_order_delivery_service",
+//path:"/"
+//}
+
+//@kubernetes:Service {
+//serviceType:"NodePort",
+//name:"ballerina-guides-phone_order_delivery_service"
+//}
+
+//@kubernetes:Deployment {
+//image:"ballerina.guides.io/phone_store_service:v1.0",
+//name:"ballerina-guides-phone_order_delivery_service"
+//}
+
+//endpoint http:Listener listener {
+//port:9091
+//};
+
+//@http:ServiceConfig {basePath:"/phonestore1"}
+//service<http:Service> phone_store_service bind listener {
+
+//Deploying on docker
+
+// import ballerinax/docker;
+// Other imports
+
+// Type definition for a  Deliver order
+
+//json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+//@docker:Config {
+//registry:"ballerina.guides.io",
+//name:"phone_order_delivery_service",
+//tag:"v1.0"
+//}
+
+// Service endpoint
+//@docker:Expose{}
+//endpoint http:Listener listener {
+//port:9091
+//};
+
+// phone store service, which allows users to order phones online for delivery
+//@http:ServiceConfig {basePath:"/phonestore1"}
+//service<http:Service> phone_order_delivery_service bind listener {
+
+type phoneDeliver record {
+    string customerName;
+    string address;
+    string contactNumber;
+    string deliveryPhoneName;
+};
+
+json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+jms:Connection jmsConnection = new({
+        initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+        providerUrl: "tcp://localhost:61616"
+    });
+
+// Initialize a queue sender using the created session
+endpoint jms:QueueSender jmsProducer2 {
+    session:jmsSession,
+    queueName:"DeliveryQueue"
+};
+
+// Initialize a JMS session on top of the created connection
+jms:Session jmsSession = new(jmsConnection, {
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"
+    });
+
+// Service endpoint
+endpoint http:Listener deliveryEP {
+    port:9091
+};
+
+@http:ServiceConfig {basePath:"/deliveryDetails"}
+
+// phone store service, which allows users to order phones online for delivery
+service<http:Service> phone_order_delivery_service bind deliveryEP {
+    // Resource that allows users to place an order for a phone
+    @http:ResourceConfig { consumes: ["application/json"],
+        produces: ["application/json"] }
+
+    sendDelivery(endpoint caller, http:Request enrichedreq) {
+        http:Response response;
+        phoneDeliver newDeliver;
+        json reqPayload;
+
+        log:printInfo(" Order Details have received from phone_store_service");
```

Comment:
extra space at the beginning, and should this be "Order details have **been** received from the phone store service" or "Order details received from the phone store service"?

Reply:
done